api/v1beta1/ovndbcluster_types.go

```diff
@@ -52,6 +52,16 @@ type OVNDBClusterSpec struct {
 	// LogLevel - Set log level info, dbg, emer etc
 	LogLevel string `json:"logLevel"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=10000
+	// OVN Northbound and Southbound RAFT db election timer to use on db creation (in milliseconds)
+	ElectionTimer int32 `json:"electionTimer"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=60000
+	// Active probe interval from standby to active ovsdb-server remote
+	ProbeIntervalToActive int32 `json:"probeInterval"`
```

Comment:
I think generally you want to have the JSON key match the name of the field.  i.e. so `json:"probeIntervalToActive"`

Reply:
Done